addons/mail/static/tests/qunit_suite_tests/components/composer_suggestion_canned_response_tests.js

```diff
@@ -11,22 +11,20 @@ QUnit.test('canned response suggestion displayed', async function (assert) {
 
     const pyEnv = await startServer();
     const mailChannelId1 = pyEnv['mail.channel'].create();
-    const { createComposerSuggestionComponent, messaging } = await start();
-    const thread = messaging.models['Thread'].findFromIdentifyingData({
-        id: mailChannelId1,
-        model: 'mail.channel',
-    });
-    const cannedResponse = messaging.models['CannedResponse'].create({
-        id: 7,
+    pyEnv['mail.shortcode'].create({
         source: 'hello',
         substitution: "Hello, how are you?",
     });
-    await createComposerSuggestionComponent(thread.composer, {
-        isActive: true,
-        modelName: 'CannedResponse',
-        recordLocalId: cannedResponse.localId,
+    const { insertText, openDiscuss } = await start({
+        discuss: {
+            params: {
+                default_active_id: mailChannelId1,
+            },
+        },
+        hasDiscuss: true,
     });
-
+    await openDiscuss();
+    await insertText('.o_ComposerTextInput_textarea', ":hello");
```

Comment:
Typo: was typing `:help` but no suggestion was shown because the canned response is actually `:hello`.